tests/update/updates_packagekit_gpk.pm

```diff
@@ -32,6 +32,14 @@ sub turn_off_screensaver {
     send_key("ctrl-d");
 }
 
+sub tell_packagekit_to_quit {
+    # tell the PackageKit daemon to stop in order to next load with new libzypp
+    # this is different from pkcon_quit
+    x11_start_program('xterm');
+    script_run("pkcon quit");
+    send_key("ctrl-d");
```

Comment:
I wonder why not just call `x11_start_program`. Then you wouldn't need to care about any opened window

Reply:
x11_start_program requires a running window in the end to look for; pkcon would not give you a window to assert on

Reply:
Well, not true. We use it also for one-shot commands